src/Orc.NuGetExplorer.Xaml/ViewModels/PackageDetailsViewModel.cs

```diff
@@ -312,7 +300,7 @@ private async Task ApplyPackageAsync()
 
                 SelectedVersion = selectedVersion;
 
-                PackageMetadataProvider = InitMetadataProvider();
+                PackageMetadataProvider = Providers.PackageMetadataProvider.CreateFromSourceContext(ServiceLocator.Default);
```

Comment:
ServiceLocator.Default or should we inject?

Reply:
I have a refactorings ready in separate branch which are get rid of this weird provider initialization, so decided to just put ServiceLocator here for now. This part should be cleared off in the near future